Userland/Libraries/LibGfx/ICC/WellKnownProfiles.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2023, Nico Weber <thakis@chromium.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibGfx/ICC/Profile.h>
+#include <LibGfx/ICC/Tags.h>
+#include <LibGfx/ICC/WellKnownProfiles.h>
+#include <time.h>
+
+namespace Gfx::ICC {
+
+static ProfileHeader rgb_header()
+{
+    ProfileHeader header;
+    header.version = Version(4, 0x40);
+    header.device_class = DeviceClass::DisplayDevice;
+    header.data_color_space = ColorSpace::RGB;
+    header.connection_space = ColorSpace::PCSXYZ;
+    header.creation_timestamp = time(NULL);
+    header.rendering_intent = RenderingIntent::Perceptual;
+    header.pcs_illuminant = XYZ { 0.9642, 1.0, 0.8249 };
+    return header;
+}
+
+static ErrorOr<NonnullRefPtr<MultiLocalizedUnicodeTagData>> en_US(StringView text)
+{
+    Vector<MultiLocalizedUnicodeTagData::Record> records;
+    TRY(records.try_append({ ('e' << 8) | 'n', ('U' << 8) | 'S', TRY(String::from_utf8(text)) }));
+    return try_make_ref_counted<MultiLocalizedUnicodeTagData>(0, 0, records);
+}
+
+static ErrorOr<NonnullRefPtr<XYZTagData>> xyzData(XYZ xyz)
```

Comment:
not entirely happy about this one too, but I guess it's some spec thing...? If `xyz_data` is understandable in this context let's use that, otherwise I guess it's fine.

Reply:
Oh hm yeah `xyz` and `XYZ` are two distinct things – but this is actually `XYZ` not `xyz`, so renamed it to `XYZ_data`…